src/directives/utils.php

```diff
@@ -1,15 +1,26 @@
 <?php
 
-function get_from_context( $expr, $context ) {
-	$path = explode( '.', $expr );
-	if ( count( $path ) > 0 && 'context' === $path[0] ) {
-		array_shift( $path );
-		$result = $context;
-		foreach ( $path as $key ) {
-			$result = $result[ $key ];
+require_once  __DIR__ . '/class-wp-directive-store.php';
+
+function store( $data ) {
+	WP_Directive_Store::merge_data( $data );
+}
+
+function evaluate( string $path, array $context = array() ) {
```

Comment:
Let's maybe add a TODO here.

```suggestion
// TODO: Implement evaluation of complex logical expressions.
function evaluate( string $path, array $context = array() ) {
```